services/minespace-web/src/tests/components/project/projectPage/ProjectPage.spec.tsx

```diff
@@ -37,7 +38,13 @@ describe("ProjectPage", () => {
   it("renders properly", () => {
     const { container } = render(
       <ReduxWrapper initialState={initialState}>
-        <ProjectPage />
+        <FeatureFlagContext.Provider
```

Comment:
We do have this mocked in all 3 setupTests files- just curious if it's not working that way or added out of habit.

Reply:
Hey, that was my mistake I added it out of habit, this is removed now.